servant-util/src/Servant/Util/Combinators/Filtering/Backend.hs

```diff
@@ -17,14 +17,14 @@ module Servant.Util.Combinators.Filtering.Backend
     , manualFilter
     ) where
 
-import Universum
+import           Universum
 
-import Data.Kind (type (*))
-import Data.Typeable (gcast1)
-import GHC.TypeLits (KnownSymbol)
+import           Data.Kind                               (Type)
+import           Data.Typeable                           (gcast1)
+import           GHC.TypeLits                            (KnownSymbol)
 
-import Servant.Util.Combinators.Filtering.Base
-import Servant.Util.Common
+import           Servant.Util.Combinators.Filtering.Base
+import           Servant.Util.Common
```

Comment:
Hm, can't believe I forgot to put `.styllish-haskell.yaml` config in this project.

Could you rebase upon the recent master and resave this file (and two below)?